string name
---
int32 num_found
geometry_msgs/PoseStamped[] poses